src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/OrganisationMinimalInfoResponse.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.professionalapi.controller.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.reform.professionalapi.domain.Organisation;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+@NoArgsConstructor
+@Getter
+public class OrganisationMinimalInfoResponse {
+
+    @JsonProperty
+    protected String name;
+    @JsonProperty
+    protected String organisationIdentifier;
+    @JsonProperty
+    protected List<ContactInformationResponse> contactInformation;
+
+    public OrganisationMinimalInfoResponse(Organisation organisation, Boolean isAddressRequired) {
+
+        getOrganisationMinimalInfoResponse(organisation, isAddressRequired);
+    }
+
+    public void getOrganisationMinimalInfoResponse(Organisation organisation, Boolean isAddressRequired) {
+        this.name = organisation.getName();
+        this.organisationIdentifier = organisation.getOrganisationIdentifier();
+
+        if (Boolean.TRUE.equals(isAddressRequired)) {
+            this.contactInformation = organisation.getContactInformation()
```

Comment:
isAddressRequired is a boolean so no need to test again with Boolean.TRUE

Reply:
Addressed here: https://github.com/hmcts/rd-professional-api/pull/694/commits/d9b19d3fcf78520dfb43772e437a7beeeb07bcb7